mozilla/fx-private-relay
# Add an animation to the custom mask tip

# New feature description

This adds an animation to the custom mask tip. It should only be visible to people visiting the website in English.

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/177217914-a165bc43-4f1b-425d-8166-eda1749c103c.png)

# How to test

[Log in with a user with a Premium account and a custom subdomain set.](https://deploy-preview-2158--fx-relay-demo.netlify.app/?mockId=full) Verify that the custom mask tip includes the animation when your browser language is English, and not when it's set to e.g. Dutch.

# Checklist

- [x] All acceptance criteria are met. ~~- I didn't see designs, so still confirming that it looks as expected.~~
